lib/generators/logidze/install/templates/migration.rb.erb

```diff
@@ -16,7 +16,7 @@ class <%= @migration_class_name %> < ActiveRecord::Migration
     end
 
     execute <<-SQL
-      CREATE OR REPLACE FUNCTION logidze_version(v bigint, data jsonb) RETURNS jsonb AS $body$
+      CREATE OR REPLACE FUNCTION logidze_version(v bigint, data jsonb, blacklist text[] DEFAULT '{}') RETURNS jsonb AS $body$
```

Comment:
When upgrading we have to drop previously defined function, 'cause we change the signature.
So, we should add here `DROP FUNCTION IF EXISTS ...` in case of upgrade (use `update?` helper).